tests/unit/actions/test_steps.py

```diff
@@ -305,6 +306,25 @@ def test_set_assignee(context_update_example: ActionContext, mocked_jira):
     mocked_jira.set_issue_status.assert_not_called()
 
 
+def test_set_assignee_failing(
+    context_create_example: ActionContext, mocked_jira, caplog
```

Comment:
Should we test the `context.operation == Operation.UPDATE` path of `maybe_assign_jira_user` as well to make sure we raise an `IncompleteStepError` in that case?

Reply:
Got it -- as long as the path has a test to make that behavior explicit 🙂